### PR TITLE
validation_6.10

### PR DESCRIPTION
**Pull request by SHANDCRUZ**
### Description
**Validated Resource**

Appliance Configuration Timeconfig
Appliance SNMPv1 Trap Destinations
Appliance SNMPv3 Users
Appliance SNMPv3 Trap Destinations
Appliance SSH Access
Appliance Device Read Community
Appliance Time and Locale Configuration
Id Pools
Id Pools Subnet
Id Pools Range
Ethernet Networks
FC Networks
FCoE Networks
Connection Templates
Network Sets
Certificates Server
Hypervisor Managers
Tasks
Scopes
OS Deployment Server
Logical Interconnect Groups
Enclosure Groups
Enclosure
Firmware Drivers
Interconnect Types
Interconnects
Logical Interconnects
Uplink Sets
Server Hardware Types
Server Hardware 
Storage Systems
Storage Pools
Storage Volume Templates
Volumes
Server Profile Templates
Hypervisor Cluster Profiles
Server Profiles
Storage Volume Attachments

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG file.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -18,407 +18,414 @@
...
+|<sub>/rest/ethernet-networks/{id}</sub>                                                  | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/ethernet-networks/{id}</sub>                                                  | PUT      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/ethernet-networks/{id}</sub>                                                  | PATCH    |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:  |
```
Comment (yuvirani) [inline endpoints-support.md:93]: Fix this

File: endpoints-support.md [hunk 2]
```diff
@@ -18,407 +18,414 @@
...
+|<sub>/rest/fc-networks</sub>                                                             | POST     |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/fc-networks/{id}</sub>                                                        | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/fc-networks/{id}</sub>                                                        | PATCH    |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:  |
```
Comment (yuvirani) [inline endpoints-support.md:102]: Same as above, change it to heavy_minus_sign

File: endpoints-support.md [hunk 3]
```diff
@@ -18,407 +18,414 @@
...
+|<sub>/rest/fcoe-networks</sub>                                                           | POST     |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/fcoe-networks/{id}</sub>                                                      | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/fcoe-networks/{id}</sub>                                                      | PATCH    |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:  |
```
Comment (yuvirani) [inline endpoints-support.md:110]: same as above

File: endpoints-support.md [hunk 4]
```diff
@@ -18,407 +18,414 @@
...
-|<sub>/rest/logical-interconnects/{id}/portFlapSettings</sub>                             | PUT      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects</sub>                                                   | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | POST     |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:  |
```
Comment (yuvirani) [inline endpoints-support.md:213]: Fix this

File: endpoints-support.md [hunk 5]
```diff
@@ -18,407 +18,414 @@
...
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | DELETE   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:  |
```
Comment (yuvirani) [inline endpoints-support.md:214]: Change this as well

File: endpoints-support.md [hunk 6]
```diff
@@ -18,407 +18,414 @@
...
+|<sub>/rest/logical-interconnects/{id}/forwarding-information-base</sub>                  | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/logical-interconnects/{id}/forwarding-information-base</sub>                  | POST     |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/logical-interconnects/{id}/forwarding-information-base/{dumpFileName}.{suffix}</sub>| GET   | :heavy_multiplication_x:   | :heavy_multiplication_x:  |  :heavy_multiplication_x:|  :heavy_multiplication_x:  | :heavy_multiplication_x:  | :heavy_multiplication_x:  | :heavy_multiplication_x:  | :heavy_multiplication_x:  | :heavy_multiplication_x:  | :white_check_mark:  |
```
Comment (yuvirani) [inline endpoints-support.md:223]: It should be heavy_multiplication_x

**Review by VenkateshRavula — APPROVED** [review]

**Review by yuvirani — APPROVED** [review]

**Pull request merged**